main.go

```diff
@@ -4296,6 +4296,9 @@ func parseModel(inputFilename string) {
 				confidentiality = model.Confidential
 			case model.StrictlyConfidential.String():
 				confidentiality = model.StrictlyConfidential
+			case "":
+				// Temporary placeholder, will later be set to `HighestConfidentiality()`
+				confidentiality = -1
```

Comment:
suggestion: Probably it is better to use something like `model.UnsetConfidentialityValue` instead of a raw int.